lib/crypto.c

```diff
@@ -64,19 +65,18 @@ crypto_init(void)
   openssl_init();
   openssl_crypto_init_threading();
 
-  if (RAND_status() < 0 || getenv("RANDFILE"))
+  if (getenv("RANDFILE"))
     {
       char rnd_file[256];
 
       RAND_file_name(rnd_file, sizeof(rnd_file));
       if (rnd_file[0])
         {
           RAND_load_file(rnd_file, -1);
+          RAND_write_file(rnd_file);
           randfile_loaded = TRUE;
+          cr_assert(RAND_status() >= 0,
```

Comment:
instead of cr_assert() use g_assert(). cr_assert is only for unit test programs.